packages/clerk-js/src/ui/components/OrganizationProfile/ProfileSettingsPage.tsx

```diff
@@ -60,6 +67,20 @@ export const ProfileSettingsPage = withCardStateProvider(() => {
       .catch(err => handleError(err, [], card.setError));
   };
 
+  const onChangeSlug = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value && !isSlug(event.target.value)) {
+      updateSlugField(event.target.value, "Slug is invalid. Can contain only lowercase characters, digits and '-'");
```

Comment:
🔧 Why are we setting this error from the FE directly?  Can we get this from the FAPI response? I don't remember if we have any other places in the code where we do input validation on the FE directly.

Also, this is a user-facing message that cannot be localised.

Reply:
@nikosdouvlis This is before we make a request(comment below). You are right about the localization.